SpatialGDK/Source/SpatialGDK/Public/Interop/SpatialClassInfoManager.h

```diff
@@ -92,6 +92,10 @@ class SPATIALGDK_API USpatialClassInfoManager : public UObject
 
 	bool TryInit(USpatialNetDriver* NetDriver, UActorGroupManager* ActorGroupManager);
 
+	// Checks whether a class is supported and quits the game if not. This is to avoid crashing
+	// when running with an out-of-date schema database.
+	bool CheckClassIsSupported(const FString& PathName);
```

Comment:
This class having a IsSupportedClass, and CheckClassIsSupported functions seems a little obtuse.
Can we rename this to indicate it's intent more clearly?
IsSupportedClass and SafeIsSupportedClass?
IsSupportedClass and ValidateOrExit_IsSupportedClass?